src/core.js

```diff
@@ -184,6 +185,10 @@ config.modules.push( config.currentModule );
 
 	// Figure out if we're running the tests from a server or not
 	QUnit.isLocal = location.protocol === "file:";
+
+	// Expose the current QUnit version
+	// `version` initialized at top of scope
+	QUnit.version = version;
```

Comment:
Since `version` isn't used anywhere else, it can be inlined, that is, use `"@VERSION"` here.